environment-dev.yml

```diff
@@ -7,8 +7,11 @@ dependencies:
   - ninja
   # Tests
   - doctest
+  # Wasm Tests
```

Comment:
Is it possible to have a condition to not get nodejs if we don't use emscripten ?

Reply:
we could introduce install node at runtime in the ci